packages/editor/src/components/collab-sidebar/comment-author-info.js

```diff
@@ -94,7 +94,7 @@ function CommentAuthorInfo( { avatar, name, date, userId } ) {
 					{ name ?? currentUserName }
 				</span>
 				{ date && (
-					<Tooltip placement="top" text={ tooltipText }>
+					<Tooltip placement="bottom" text={ tooltipText }>
```

Comment:
```suggestion
					<Tooltip text={ tooltipText }>
```

Nit: [Since the default value is "bottom"](https://github.com/WordPress/gutenberg/blob/1fae6d281dd5c4fcab22620bc0d32094193af831/packages/components/src/tooltip/README.md#placement-top--top-start--top-end--right--right-start--right-end--bottom--bottom-start--bottom-end--left--left-start--left-end), the prop itself is unnecessary.